### ISSUE TITLE
Error from colorPicker component

### ISSUE THREAD

**Issue by AlexandraBuzila** [magic-script/magic-script-components#51]
The colorPicker component throws an error on creation:

`E MagicScript: VM: Unhandled promise rejection: ReferenceError: properties is not defined`

**Comment by grozdanov** [comment]
The issue has been resolved in version 0.0.15.

**Comment by grozdanov** [comment]
Test code:
```
export default class MyApp extends React.Component {

  onCancelHandler = (event) => {
    console.log('[ColorPicker]: Cancel');
    console.log(event);
  }

  onConfirmHandler = (event) => {
    console.log('[ColorPicker]: Confirm');
    console.log(event);
  }

  onColorChangedHandler = (event) => {
    console.log('[ColorPicker]: Color Changed');
    console.log(event);
  }

  render() {
    return (
      <view name='main-view'>
        <colorPicker
          localPosition={[0, 0, 0]}
          color={[0.1, 0.3, 0.1, 0.84]}
          height={0.50}
          onCancel={this.onCancelHandler}
          onConfirm={this.onConfirmHandler}
          onColorChanged={this.onColorChangedHandler}
        ></colorPicker>
      </view>
    );
  }
}
```
